app/models/customer.rb

```diff
@@ -75,6 +75,36 @@ def require_driver_form?
     !driver_form_content.blank?
   end
 
+  # returns Human readable string of the validity length of driver training
+  def helper_driver_training_validity_length?
```

Comment:
Good points @mitchellharper12. 

@aayush19saxena take a look at http://apidock.com/rails/ActionView/Helpers/DateHelper/distance_of_time_in_words for some nice techniques for solving these problems. Click "Show source" to view the source for that method. Take particular note of the use of `case` and `when` and the use of ranges (e.g., `2...45`) instead of the if-elsif-else structure.